src/app/room.service.ts

```diff
@@ -0,0 +1,101 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+import { Observable, of } from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+
+import { Room } from './Room';
+
+
+@Injectable({ providedIn: 'root' })
+export class RoomService {
+
+  private roomsUrl = 'api/rooms';  // URL to web api
+
+  httpOptions = {
+    headers: new HttpHeaders({ 'Content-Type': 'application/json' })
+  };
+
+  constructor(
+    private http: HttpClient) { }
```

Comment:
on same line with 19, remove space in {}

Reply:
Done